### DISCUSSION TITLE
Text with color gradient

### DISCUSSION THREAD

**Discussion by KBLiveSolutions** [Bodmer/TFT_eSPI#3542] [Q&A - Fonts]
Is it possible to have shaded text ?

I have tried the Smooth_font_gradient sketch, but it doesn't exactly do that.
Any idea on how to achieve this?

**Comment by MYCAMEL222** [comment]
Are you looking to do "drop shadow text" like this..........

![20240705_120949](https://github.com/user-attachments/assets/ec4ac4e6-b798-4dc3-b972-ce7588c6be03)
